docs/guides/_toc.json

```diff
@@ -125,83 +125,73 @@
-                },
-                {
-                  "title": "Specify observables in the Pauli basis",
-                  "url": "/guides/specify-observables-pauli"
-                },
-                {
-                  "title": "The Operator class",
-                  "url": "/guides/operator-class"
-                }
-              ]
-            },
-            {
-              "title": "Other circuit building tools",
-              "children": [
-                {
-                  "title": "Pulse schedules",
-                  "url": "/guides/pulse"
-                },
-                {
-                  "title": "OpenQASM",
-                  "children": [
-                    {
-                      "title": "Intro to OpenQASM",
-                      "url": "/guides/introduction-to-qasm"
-                    },
-                    {
-                      "title": "OpenQASM 2 and the Qiskit SDK",
-                      "url": "/guides/interoperate-qiskit-qasm2"
-                    },
-                    {
-                      "title": "OpenQASM 3 and the Qiskit SDK",
-                      "url": "/guides/interoperate-qiskit-qasm3"
-                    },
-                    {
-                      "title": "OpenQASM 3 feature table",
-                      "url": "/guides/qasm-feature-table"
-                    },
-                    {
-                      "title": "OpenQASM 3.x live specification",
-                      "url": "https://openqasm.com/"
-                    }
-                  ]
+              {
+                "title": "Circuit library",
+                "url": "/guides/circuit-library"
+              },
+              {
+                "title": "Construct circuits",
+                "url": "/guides/construct-circuits"
+              },
+              {
+                "title": "Classical feedforward and control flow",
+                "url": "/guides/classical-feedforward-and-control-flow"
+              },
+              {
+                "title": "Synthesize unitary operators",
+                "url": "/guides/synthesize-unitary-operators"
+              },
+              {
+                "title": "Bit-ordering in the Qiskit SDK",
+                "url": "/guides/bit-ordering"
+              },
+              {
+                "title": "Save circuits to disk",
+                "url": "/guides/save-circuits"
+              },
+              {
+                "title": "Operators module overview",
+                "url": "/guides/operators-overview"
+              },
+              {
+                "title": "Specify observables in the Pauli basis",
+                "url": "/guides/specify-observables-pauli"
+              },
+              {
+                "title": "The Operator class",
+                "url": "/guides/operator-class"
+              }
+            ]
+          },
+          {
+            "title": "Other circuit-building tools",
```

Comment:
the "other circuit building tools section" should still be inside the circuits and operators top level section

Reply:
oops! Yes indeed, will fix